docs/src/main/paradox/project/scala3.md

```diff
@@ -1,15 +1,9 @@
 # Scala 3 support
 
-Pekko has experimental support for Scala 3.
+Pekko has support for Scala 3. Now that the Scala 3.3 LTS release is out, we can be pretty confident that it is unlikely that we will need to make binary incompatible changes due to Scala 3 changes. See the @ref:[binary compatibility documentation](../common/binary-compatibility-rules.md).
```

Comment:
I think we can also remove the "Now" phrasing in "now that the Scala 3.3 LTS release". Just mention that Pekko is built against Scala 3.3 LTS and because of that we don't forsee any breaking binary compatibility issues in the foreseeable future.

Reply:
reworded the text